ios/FluentUI/Table View/TableViewCell.swift

```diff
@@ -58,6 +58,25 @@ public enum TableViewCellAccessoryType: Int {
     }
 }
 
+// Different background color is shown based on if `TableViewCell` is shown in `TableView` as grouped or plain style
+@objc(MSFTableViewCellBackgroundStyleType)
+public enum TableViewCellBackgroundStyleType: Int {
+    case plain
+    case grouped
+    case clear
+
+    var defaultColor: UIColor {
+        switch self {
+        case .plain:
```

Comment:
Should I replace these values with tokens later?

Reply:
i have a different draft